src/components/ProductImage/ProductImage.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-invalid-this */
+import React, { Component } from 'react';
+import './ProductImage.scss';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+export const CN = 'product-image';
+
+class ProductImage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedImage: '',
+    };
+  }
+
+  clickHandler = (e) => {
+    const selectedImage = e.currentTarget.dataset.src;
```

Comment:
why don't u path src or image id like parameter instead of use `e.currentTarget.dataset.src` ?